### DISCUSSION TITLE
Why is such a translation approved. 阿登侧边栏

### DISCUSSION THREAD

**Discussion by uiolee** [mdn/mdn-community#402] [Localization] [answered]
![图片](https://github.com/mdn/mdn-community/assets/22849383/2336c923-290a-4927-bfab-4a6374ea5d5f)

Just sharing, no offense.

**Comment by fwqaaq** [comment]
It has for more than just these errors, perhaps it could be a issue from a long time ago, and you may be able to fix it.

**Comment by yin1999** [comment, marked as answer]
The [broken "sidebar" was already there](https://github.com/mdn/translated-content/blob/33058f2b292b3a581333bdfb21b8f671898c5060/files/zh-cn/mozilla/add-ons/webextensions/api/tabs/%E6%9F%A5%E8%AF%A2/index.html) when we [migrated to GitHub](https://hacks.mozilla.org/2020/12/welcome-yari-mdn-web-docs-has-a-new-platform/). We could not track the revisions when those content are served in old Kuma, so I can't tell the reason. 

> I'm in the process of fixing it.

Thanks for catching this and working to fix it :)

**Comment by yin1999** [comment]
This is fixed in mdn/translated-content#14086 :)

**Reply by uiolee** [reply]
thanks. i am sorry that i make a promise. but i want to explain that i have modified it, but have not create a PR
